kenjutsu/format.py

```diff
@@ -86,7 +86,7 @@ def reformat_slice(a_slice, a_length=None):
             new_slice.append(reformat_slice(i, a_length))
         return new_slice
     elif not isinstance(a_slice, slice):
-        raise ValueError(
+        raise TypeError(
             "Expected a `slice` type. Instead got `%s`." % str(a_slice)
```

Comment:
This message is pretty outdated and should be updated to specify the currently supported types.